app/views/MainTemplate.scala.html

```diff
@@ -192,7 +193,7 @@
 }
 
 @templateArguments = @{
-    if(!isUserResearchBannerHidden && appConfig.urBannerEnabled) {
+    if(!isUserResearchBannerHidden && (appConfig.urBannerEnabled || showUrBannerOverride)) {
```

Comment:
is it correct to ignore the appConfig if a message sets showUrBannerOverride to true?

Reply:
yeah this is the functionality we want as we want to be able to show the banner on one page only